src/main.rs

```diff
@@ -308,6 +308,19 @@ fn main() -> BError {
                 Effectors::DoubleDamage,
             ],
         ),
+        SkillDefinition::new(
+            Skills::DoubleAttackSpeed,
+            String::from("Double Attack Speed"),
+            String::from("double_attack_speed"),
+            String::from("Double the attack speed."),
+            0.0,
```

Comment:
If we decide to keep the item but have a cooldown, this is where we should set it. :)